packages/elements/src/photon-multirx-form/photon-prescribe-workflow.tsx

```diff
@@ -164,9 +169,9 @@ export function PrescribeWorkflow(props: PrescribeProps) {
   });
   createEffect(() => {
     if (authenticated()) {
-      const enableNewMedicationSearch = newMedSearchTesters.includes(
-        client?.authentication.state.user?.name
-      );
+      const enableNewMedicationSearch =
+        newMedSearchTesters.includes(client?.authentication.state.user?.name) ||
+        newMedSearchOrgs.includes(client?.authentication.state.user?.org_id);
```

Comment:
Not to be annoying, but can we put this in `package/settings`?

Reply:
Don't know that it can, elements doesnt seem to ahve access to `@client/settings`

Reply:
This would only apply to customers who embed with elements. Piction for example looks to be using our site, so  the way we typically did this in the past is we did this for them was to add a parameter to the prescribe flow https://github.com/Photon-Health/client/blob/00d6a407467f0938f842f66c58678bd5297f05ff/packages/elements/src/photon-multirx-form/photon-prescribe-workflow-component.tsx#L30C9-L30C25

For features that will be requirements, we just asked them to bump the version